mtags/src/main/scala-2/scala/meta/internal/pc/completions/AmmoniteCompletions.scala

```diff
@@ -152,18 +152,19 @@ trait AmmoniteCompletions { this: MetalsGlobal =>
                 pos.withStart(rangeStart).withEnd(pos.point).toLsp
               }
 
-            (javaCompletions ++ scalaCompletions).map { c =>
-              new TextEditMember(
-                filterText = c,
-                edit = new l.TextEdit(
-                  ivyEditRange,
-                  if (isInitialCompletion) s"`$c$$0`" else c
-                ),
-                sym = select.symbol
-                  .newErrorSymbol(TermName("artefact"))
-                  .setInfo(NoType),
-                label = Some(c.stripPrefix(":"))
-              )
+            (javaCompletions ++ scalaCompletions).zipWithIndex.map {
+              case (c, index) =>
+                new TextEditMember(
+                  filterText = c,
+                  edit = new l.TextEdit(
+                    ivyEditRange,
+                    if (isInitialCompletion) s"`$c$$0`" else c
+                  ),
+                  sym = select.symbol
+                    .newErrorSymbol(TermName(s"artefact$index"))
```

Comment:
Somehow the symbols were found to be the same and deduplicated. Not sure why this didn't happen in Ammonite